Config/Database.go

```diff
@@ -1,37 +1,46 @@
-//Config/Database.go
-package Config
+package config
```

Comment:
you can rename this package to db, since it is only exposing db specific config as of now.